spec/models/customer_spec.rb

```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe Customer, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```

Comment:
this will need to be filled in later with validations. do you think we want to validate for uniqueness of names or just that the name is present?